appscale/tools/local_state.py

```diff
@@ -420,6 +420,29 @@ def update_local_metadata(cls, options, db_master, head_node):
 
 
   @classmethod
+  def clean_local_metadata(cls, keyname):
+    try:
+      with open(cls.get_locations_json_location(keyname), 'r') as file_handle:
+        file_contents = yaml.safe_load(file_handle.read())
```

Comment:
String comparisons between unicode and non-unicode strings should work just fine.
```
>>> u'value' == 'value'
True
```
Standard libs should be preferred over 3rd party ones in most cases unless they make the code more difficult to maintain.